src/main/java/org/gitlab/api/GitlabAPI.java

```diff
@@ -585,8 +585,15 @@ public GitlabProject getProject(Serializable projectId) throws IOException {
      * use namespace & project name to get project
      */
     public GitlabProject getProject(String namespace, String projectName) throws IOException{
-        String tailUrl = GitlabProject.URL + "/" + namespace + "%2F" + projectName;
-        return retrieve().to(tailUrl, GitlabProject.class);
+        String fullName = namespace+" / "+projectName;
```

Comment:
I tested on our internal GitLab and we really have a space between each side of the slash. It's working with the changes.

However, may this be a GitLab configuration ?